libs/content-search-toolkit/README.md

```diff
@@ -1,7 +1,4 @@
 # content-search-toolkit
```

Comment:
After rebasing with main, you can remove the `<!-- gitbook-ignore -->` and re-run `yarn gitbook` to include this readme to the docs